accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -1127,6 +1127,12 @@ A compile-time error occurs if any direct or indirect superinterface of
 _DV_ denotes the declaration _DV_. *As usual, subtype cycles are not
 allowed.*
 
+It is a compile-time error if two elements in the type list of
+the `implements` clause of an extension type specifies the same type.
+
+*This rule against duplicates is identical to the existing rule about 
```

Comment:
It's not *identical* (it contains different words, like "extension type"), but it does serve the same purpose.

I'd change "about" here to "for". Mainly a feeling. The word "about" is vague.

Consider:

> *This rule against duplicate types mirrors the similar rule for class, mixin and enum declarations.*